bitrise.yml

```diff
@@ -35,85 +35,99 @@ workflows:
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
         title: Run Lint for SDK
     - android-unit-test@1:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$WIDGET_SDK_MODULE"
-        - variant: "$SDK_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
         title: Unit Test for SDK
     - android-lint@0:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$EXAMPLE_APP_MODULE"
-        - variant: "$INTEGRATOR_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
         title: Run Lint for APP
     - android-unit-test@1:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$EXAMPLE_APP_MODULE"
-        - variant: "$INTEGRATOR_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
         title: Unit Test for APP
     - deploy-to-bitrise-io@1:
         inputs:
-        - notify_email_list: "$BUILD_EMAILS"
+        - notify_email_list: $BUILD_EMAILS
     - cache-push@2: {}
     envs:
     - opts:
         is_expand: false
       INTEGRATOR_VARIANT: master
   increment_project_version:
     steps:
-    - activate-ssh-key@4:
-        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
-    - git-clone@6: {}
-    - cache-pull@2: {}
-    - install-missing-android-tools@3:
-        inputs:
-        - gradlew_path: "$PROJECT_LOCATION/gradlew"
     - gradle-runner@2:
         inputs:
-        - gradle_file: "$PROJECT_LOCATION/build.gradle"
-        - gradlew_path: "$PROJECT_LOCATION/gradlew"
-        - gradle_task: ":bumpVersionMinorLevel"
+        - gradle_file: $PROJECT_LOCATION/build.gradle
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+        - gradle_task: ':bumpVersionMinorLevel'
     - script@1:
         inputs:
-        - content: |-
+        - content: >-
             #!/usr/bin/env bash
+
             # fail if any commands fails
+
             set -e
-            # make pipelines' return status equal the last command to exit with a non-zero status, or zero if all commands exit successfully
+
+            # make pipelines' return status equal the last command to exit with
+            a non-zero status, or zero if all commands exit successfully
+
             #set -o pipefail
+
             # debug log
+
             set -x
 
+
             NEW_VERSION=`./gradlew -q printCurrentVersionName`
+
             BRANCH_NAME="project-version-increment/${NEW_VERSION}"
+
             MESSAGE="Increment project version to ${NEW_VERSION}"
 
+
             git checkout -b $BRANCH_NAME
+
             git add -A
+
             git commit -m "$MESSAGE"
+
             git push origin "$BRANCH_NAME":"$BRANCH_NAME"
 
+
```

Comment:
Same in few other 'script' steps

Reply:
I also thought of that, but the issue is that every time we make a change in Bitrise and want to copy these changes back to the repository, it will happen again and then we would need to remove this changes constantly. So it will start to become a chore to fix this manually every time, since there's quite a lot of these situations.